docs/guides/deploy-guide/examples/testbed.md

```diff
@@ -13,30 +13,35 @@ OSISM is the reference implementation for the infrastructure as a service layer
 used in the SCS project to test and work on the Instrastructure as a Service layer.
 
 The OSISM Testbed is intended as a playground. Further services and integration will
-be added over time. More and more best practices and experiences from the productive
+be added over time. A increasing number of best practices and experiences from the productive
 deployments will be included here in the future. It will become more production-like
 over time. However, at no point does it claim to represent a production setup exactly.
 
 ## Requirements
 
 ### Cloud access
 
-The prerequisite is to have an account on one of the supported OpenStack cloud providers.
+The usual prerequisite is to have an account on one of the supported OpenStack cloud providers.
+
+In principle, it is also possible to add further environments on which the testbed can be operated 
+with manageable effort: Virtual machines, network and storage based on OpenStack are used as the basis or to
+provide the testbed. As OSISM/SCS also virtualizes systems itself, the OpenStack environment must provide 
```

Comment:
I would prefer "SHOULD" or "RECOMENDED" instead of "MUST" here. When you do not want to run payload on top of the testbed it's totally fine to not have nested virtualisation. E.g. you want to test the upgrade of the HAProxy oder RabbitMQ or something like this.

Reply:
Done